crates/re_types/src/components/view_coordinates_ext.rs

```diff
@@ -6,7 +6,16 @@ use crate::view_coordinates::{Axis3, Handedness, Sign, SignedAxis3, ViewDir};
 
 use super::ViewCoordinates;
 
+impl Default for ViewCoordinates {
+    fn default() -> Self {
+        Self::DEFAULT
+    }
```

Comment:
I think this is a food-gun waiting to go off. Better to be explicit, especially since we reuse `ViewCoordinates` for too many things:

* https://github.com/rerun-io/rerun/issues/2663

Reply:
food gun 😋